db/keyval/redis/bytes_broker_impl.go

```diff
@@ -187,169 +178,87 @@ func (db *BytesConnectionRedis) ListValues(match string) (keyval.BytesKeyValIter
 	return &bytesKeyValIterator{values: kvs}, nil
```

Comment:
is it possible to improve the implementation in term of cursor? (is it different story?)